pkg/controller.v2/trainjob_controller.go

```diff
@@ -36,6 +39,15 @@ import (
 
 var errorUnsupportedRuntime = errors.New("the specified runtime is not supported")
 
+type objsOpState int
+
+const (
+	succeeded      objsOpState = iota
```

Comment:
Let's call it `creationSucceeded` to be more explicit ?
```suggestion
	creationSucceeded      objsOpState = iota
```

Reply:
Sure

Reply:
Done.